### PR TITLE
Add paper "Discovery of a Planar Black Hole Mass Scaling Relation for Spiral Galaxies"

### PR DESCRIPTION
**Pull request by ZehaoJin**
I uploaded both a .gif animation and a .pdf plot. Use the plot instead of animation doesn't work.

### CODE REVIEW

**Comment by MilesCranmer** [comment]
Hi @ZehaoJin,
Thanks so much for submitting this! Unfortunately the gif file is 10 MB which would increase the repo size by quite a bit will slow down the page loading times. Could you please only include the image? I think you can convert the pdf to an svg which will also be vectorized and small in size. (But PDFs will not easily display on the web page)
Thanks!
Miles

**Comment by ZehaoJin** [comment]
Thank you very much! I put a png image instead, it should work now.

**Comment by ZehaoJin** [comment]
@MilesCranmer Thank you very much! I put a png image instead, it should work now.

**Comment by MilesCranmer** [comment]
Thanks. Do you know what this issue is caused by? 

https://results.pre-commit.ci/run/github/295391759/1696357984.nzf76DElTmOZ08A51kO8tw

once all the tests pass I can merge

**Comment by ZehaoJin** [comment]
@MilesCranmer Thanks for pointing me to the error! Now seems like it passes

File: docs/papers.yml [hunk 1]
```diff
@@ -2,6 +2,16 @@
 # information to generate the "Research Showcase"
 
 papers:
+  - title: Discovery of a Planar Black Hole Mass Scaling Relation for Spiral Galaxies
+    authors:
+      - Benjamin L. Davis (1)
+      - Zehao Jin (1)
+    affliations:
```
Comment (MilesCranmer) [inline docs/papers.yml:9]:
```suggestion
    affliations:
```

I think this is the last remaining issue. I can fix this myself though!

**Comment by MilesCranmer** [comment]
Thanks again for submitting. Very exciting result and paper by the way!! Let me know if you posted it on twitter or LinkedIn and I'm happy to help share it more widely. It is a super nice application!

**Pull request merged**